pkg/server/server.go

```diff
@@ -157,12 +157,6 @@ func (s *Server) StreamedListObjects(req *openfgapb.StreamedListObjectsRequest,
 
 	typesys := typesystem.New(model)
 
-	for _, ctxTuple := range req.GetContextualTuples().GetTupleKeys() {
```

Comment:
This is in https://github.com/openfga/openfga/blob/main/pkg/server/commands/list_objects.go#L66